client/src/modules/invoices/registry/search.modal.js

```diff
@@ -12,29 +12,34 @@ InvoiceRegistrySearchModalController.$inject = [
  * This controller is responsible to collecting data from the search form and modifying
  * the underlying filters before passing them back to the parent controller.
  */
-function InvoiceRegistrySearchModalController(ModalInstance, Services, filters, Notify, Store, Periods, util) {
+function InvoiceRegistrySearchModalController(ModalInstance, filters, Notify, Store, Periods, util, Invoices) {
   var vm = this;
   var changes = new Store({ identifier : 'key' });
   vm.filters = filters;
 
   vm.defaultQueries = {};
 
+  // displayValues will be an id:displayValue pair
+  var displayValues = {};
+
+  var lastViewFilters = Invoices.filters.formatView().customFilters;  
+  
+  // map key to last display value for lookup in loggedChange
+  var lastDisplayValues = lastViewFilters.reduce(function (object, filter) {
```

Comment:
In the future, we should look at making this into a function - maybe in `FilterService`.  Something like....
```js
var lastDisplayValues = Invoices.filters.getDisplayMap();
```